core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -55,14 +56,15 @@ public class ClusterConfig {
     private final Map<Integer, Map<String, String>> perBrokerOverrideProperties;
 
     @SuppressWarnings("checkstyle:ParameterNumber")
-    private ClusterConfig(Type type, int brokers, int controllers, String name, boolean autoStart,
+    private ClusterConfig(Type type, int brokers, int controllers, int disksPerBroker, String name, boolean autoStart,
                   SecurityProtocol securityProtocol, String listenerName, File trustStoreFile,
                   MetadataVersion metadataVersion, Map<String, String> serverProperties, Map<String, String> producerProperties,
                   Map<String, String> consumerProperties, Map<String, String> adminClientProperties, Map<String, String> saslServerProperties,
                   Map<String, String> saslClientProperties, Map<Integer, Map<String, String>> perBrokerOverrideProperties) {
         this.type = Objects.requireNonNull(type);
         this.brokers = brokers;
         this.controllers = controllers;
+        this.disksPerBroker = disksPerBroker;
```

Comment:
Could you add check to make sure the value is larger than zero?

Reply:
I add check in `ClusterConfig#Builder#build` function.